apps/docs/src/app/core/component-docs/calendar/examples/calendar-disabled-navigation-buttons-example/calendar-disabled-navigation-buttons-example.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-calendar-disabled-navigation-buttons-example',
+    templateUrl: 'calendar-disabled-navigation-buttons-example.html'
+})
+export class CalendarDisabledNavigationButtonsExampleComponent {
+    /**
+     * Function used to disable previous button in the calendar header.
+     */
+    previousButtonDisableFunction(): boolean {
```

Comment:
You should add some logic, so it's clear how to use these functions. Would also be nice to see different use cases for prev and next buttons. And please, use typed arrow functions:
```typescript
import { DisableHeaderButtonFunc } from '@fundamental-ngx/core/calendar';
...
export class ExampleComponent {

previousButtonDisableFunction: DisableHeaderButtonFunc = () => { .... };
nextButtonDisableFunction: DisableHeaderButtonFunc = () => { .... };

```